infra/hooks/postdeploy.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+echo "Configuring EventGrid subscription for API Center"
+
+# Sleep for 2 minutes to allow the function app to be deployed
+sleep 120
+
+functionID=$(az functionapp function show --name ${AZURE_FUNCTION_NAME} \
+    --function-name apicenter-analyzer --resource-group ${RESOURCE_GROUP_NAME} \
+    --query "id" --output tsv)
+MSYS_NO_PATHCONV=1 az eventgrid event-subscription create --name MyEventSubscription \
+    --source-resource-id "${AZURE_API_CENTER_ID}" --endpoint "$functionID" \
+    --endpoint-type azurefunction --included-event-types \
+    Microsoft.ApiCenter.ApiDefinitionAdded Microsoft.ApiCenter.ApiDefinitionUpdated
```

Comment:
Instead of giving it a name of `MyEventSubscription`, I'd more towards having a meaningful name for the event grid subscription.

Because this sub is only looking for both `Microsoft.ApiCenter.ApiDefinitionAdded` and `Microsoft.ApiCenter.ApiDefinitionUpdated`, can we give it a name like `on-api-definition-added-or-updated`?